cbmc_viewer/make_metric_report.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# -*- mode: python-mode -*-
+
+"""Generate CBMC runtime analysis metrics report."""
+
+
+import argparse
+import sys
+
+from cbmc_viewer import metric_reportt
+from cbmc_viewer import optionst
+
+def create_parser():
+    """Command line parser."""
+
+    parser = argparse.ArgumentParser(
+        description="Generate CBMC runtime analysis metrics report."
+    )
+
+    optionst.srcdir(parser)
+    optionst.reportdir(parser)
+
+    parser.add_argument(
```

Comment:
These are the arguments I recommended moving into optionst.py

Reply:
Done.